org.eclipse.scout.rt.jackson/src/main/java/org/eclipse/scout/rt/jackson/dataobject/DefaultDoEntityDeserializerTypeStrategy.java

```diff
@@ -48,7 +49,9 @@ public String resolveTypeVersion(Class<? extends IDoEntity> entityClass) {
   @Override
   public void putContributions(IDoEntity doEntity, String attributeName, Collection<?> contributions) {
     // add contributions to corresponding list in do entity
-    //noinspection unchecked
-    doEntity.getContributions().addAll((Collection<? extends IDoEntityContribution>) contributions);
+    if (!CollectionUtility.isEmpty(contributions)) {
```

Comment:
Add this check also in raw case?
org.eclipse.scout.rt.jackson.dataobject.RawDoEntityDeserializerTypeStrategy#putContributions